OSDOCS-11795: Shared VPC doc updates. (#87731)
openshift/openshift-docs: modules/rosa-sharing-vpc-hosted-zones.adoc

  ]
}
----

> . Create a private hosted zone in the link:https://us-east-1.console.aws.amazon.com/route53/v2/[Route 53 section of the AWS console]. In the hosted zone configuration, the domain name is `<cluster_domain_prefix>.<reserved_dns_domain>`. The private hosted zone must be associated with the created VPC.

Reviewer: ```suggestion
. Create a private hosted zone in the link:https://us-east-1.console.aws.amazon.com/route53/v2/[Route 53 section of the AWS console]. Configure the hosted zone with the following details:

* The domain name as `<cluster_domain_prefix>.<reserved_dns_domain>`
* The private hosted zone associated with the created VPC.
```

Author: This is outside of the scope of the Jira ticket, though I will make a new ticket to revise the formatting.

Author: OSDOCS-13319 has been created so that the formatting issues can be addressed. Thank you!
